guides/common/modules/proc_downloading-python-packages-to-a-host-from-a-python-repository.adoc

```diff
@@ -1,5 +1,5 @@
 [id="Downloading_Files_to_a_Host_from_a_Python_Type_Repository_{context}"]
-= Downloading Files to a Host From a Python Type Repository
+= Downloading files to a host from a python type repository
```

Comment:
```suggestion
= Downloading files to a host from a python type repository
```

unsure here; IMHO "Python" is a name; but "python" might also be the name of the content type in Katello. :shrug:

Reply:
Agree. I think Python should be capitalized but it is consistently lower-case in Foreman.

Reply:
Not changing because I would have to update the text as well.